tests/plugins/copyformatting/manual/focus.md

```diff
@@ -1,4 +1,4 @@
-@bender-tags: bug, 4.7.0, trac16845, copyformatting
+@bender-tags: bug, 4.7.0, 4.9.0, trac16845, copyformatting, 1458
 @bender-ui: collapsed
 @bender-ckeditor-plugins: copyformatting,toolbar,wysiwygarea,floatingspace,elementspath,image2,widget,justify,div
```

Comment:
Same here, you only need `copyformatting,toolbar,wysiwygarea,image2,justify,div` (I know this tests was not added by this PR, but it is a good opportunity to fix it:D).